app/controllers/publishers/vacancies/job_applications_controller.rb

```diff
@@ -54,12 +54,9 @@ def job_application
   end
 
   def status
-    case params[:commit]
-    when t("buttons.shortlist")
-      "shortlisted"
-    when t("buttons.confirm_rejection")
-      "unsuccessful"
-    end
+    return "shortlisted" if params[:publishers_job_application_update_status_form].key?("further_instructions")
```

Comment:
This doesn't feel that abstract - is it possible to test the action_name or the path instead?

Reply:
the action_name is the same `update_status`